schemas/contact_number.py

```diff
@@ -0,0 +1,18 @@
+from ma import ma
+from models.contact_number import ContactNumberModel
+from utils.time import time_format
+from marshmallow import fields, post_load, EXCLUDE
+
+
+class ContactNumberSchema(ma.SQLAlchemyAutoSchema):
+    class Meta:
+        model = ContactNumberModel
+        unknown = EXCLUDE
```

Comment:
It's not necessary, just an artifact from when I was trying different things that I forgot about, I will remove it! FWIW, I found in the marshmallow docs that it can be set in the Meta class, when a schema is initialized (i.e. ContactNumberSchema(unknown=INCLUDE/EXCLUDE), or passed as in argument to load()

Reply:
Nice! That is good to know.